gcp_variant_transforms/libs/bigquery_vcf_schema.py

```diff
@@ -120,9 +122,12 @@ def generate_schema_from_header_fields(
       continue
     calls_record.fields.append(bigquery.TableFieldSchema(
         name=bigquery_util.get_bigquery_sanitized_field_name(key),
-        type=bigquery_util.get_bigquery_type_from_vcf_type(field.type),
-        mode=_get_bigquery_mode_from_vcf_num(field.num),
-        description=bigquery_util.get_bigquery_sanitized_field(field.desc)))
+        type=bigquery_util.get_bigquery_type_from_vcf_type(
+            field[_HeaderKeyConstants.TYPE]),
```

Comment:
Why is this change needed? Can't you continue to use field.type, field.num, etc.?

Reply:
Unlike HeaderFields that used the namedtuples from pyvcf directly, we convert them to dicts in VcfHeader (there was an issue with picking namedtuples, so we had to convert them to dicts). So, the field no longer has a `num` attribute, but just a key. This will be nicer when we switch to using Nucleus protos as those have properly named fields.